### PR TITLE
add jackson-module-parameter-names

### PR DESCRIPTION
**Pull request by Sam-Kruglov**
couldn't watch you add those annotations everywhere :D

### CODE REVIEW

File: src/test/java/io/axoniq/demo/gamerental/controller/GameRentalRestControllerTest.java [hunk 1]
```diff
@@ -34,7 +38,18 @@ void setUp() {
         commandGateway = mock(ReactorCommandGateway.class);
         queryGateway = mock(ReactorQueryGateway.class);
 
-        testClient = WebTestClient.bindToController(new GameRentalRestController(commandGateway, queryGateway)).build();
+        testClient = WebTestClient.bindToController(new GameRentalRestController(commandGateway, queryGateway))
+                .httpMessageCodecs(this::jsonCodecs)
+                .configureClient().codecs(this::jsonCodecs)
+                .build();
```
Comment (Sam-Kruglov) [inline src/test/java/io/axoniq/demo/gamerental/controller/GameRentalRestControllerTest.java:44]: it also might pick up that module if you build `WebTestClient` from spring's `@AutoconfigureWebTestClient` but it's probably slower

**Comment by smcvb** [comment]
Haha, thanks, Sam. I did it on purpose to make the intent clearer, but I agree this is cleaner. 
I think I'll use it for my upcoming live coding session!

No offense btw, but I won't take the PR as is.
I want to push the adjustments to the correct branches, correlating to the steps they're in.
That will keep the flow of changes identical when someone jumps through them.
I'll push the commit messages with creds to your GitHub account, though!

**Comment by Sam-Kruglov** [comment]
Sure, don’t bother with credits, I just added a dependency :)

**Comment by smcvb** [comment]
Gotcha Sam!

**Comment by Sam-Kruglov** [comment]
you could also have a master branch with N commits instead of N branches. That way you can still walk through the steps using git hard reset but you can also see the diff. You can compare branches but that's non-trivial. I did it for my presentation https://github.com/Sam-Kruglov/bike-rental-demo/commits/master

**Comment by smcvb** [comment]
> you could also have a master branch with N commits instead of N branches. That way you can still walk through the steps using git hard reset but you can also see the diff. You can compare branches but that's non-trivial. And it's probably easier to manage. I did it for my presentation https://github.com/Sam-Kruglov/bike-rental-demo/commits/master

Thanks, but I intended it this way.

**Comment by smcvb** [comment]
I only now notice that this PR is still open...
Thanks again for providing it, @Sam-Kruglov, but I will be closing it.

As stated earlier, I wanted to showcase it with annotations on purpose.
Not because it is the best solution but to make it overly clear _something_ needs to be done.
By repeating the annotation, that something becomes clearer than with a single dependency, in my honest opinion.

The current format of this sample project uses `@ConstructorProperties`, by the way.

**Pull request closed without merging**